.drone.yml

```diff
@@ -13,17 +13,6 @@ resources:
     memory: 2GiB
 
 steps:
-  - name: restore
-    image: plugins/s3-cache
-    settings:
-      restore: true
-      endpoint:
-        from_secret: CACHE_S3_ENDPOINT
-      access_key:
-        from_secret: CACHE_S3_ACCESS_KEY
-      secret_key:
-        from_secret: CACHE_S3_SECRET_KEY
-
   - name: test
     image: golangci/golangci-lint:v1.56.0
     environment:
```

Comment:
nit: this isn't really necessary without the caching.

Reply:
removed them